Android/JavaxInject/cgmanifest.json

```diff
@@ -0,0 +1,16 @@
+{
+  "Registrations":[ 
+    {
+      "Component": {
+        "Type": "Maven",
+        "Maven": {
+          "ArtifactId": "dagger",
+          "GroupId": "com.google.dagger",
+          "Version": "2.13",
+          "NuGetId": "Xamarin.Google.Dagger"
```

Comment:
These should be the javax.inject names and versions.

Reply:
Got it.